Validation/RecoTrack/python/JetCoreMCtruthSeedGenerator_cfi.py

```diff
@@ -0,0 +1,15 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
this python file is redundant, `scram b` generates it automatically from the fillDescriptions. plese remove it.

Reply:
I'm confused. I removed the file, I run scram b, but I don't see the recreated file. can I still do
`import RecoTracker.TkSeedGenerator.DeepCoreSeedGenerator_cfi`
from a non-existing cfi?

Reply:
The file is put under `$CMSSW_BASE/cfipython/.../` by `scram b` (it was bit confusing at first for me as well). the logic is that if you can, it makes sense to define the parameters just once, in the fillDescriptions.

Reply:
Am I doing something wrong with the description? The file is not here

```
ll $CMSSW_BASE/cfipython/slc7_amd64_gcc820/RecoTracker/TkSeedGenerator/
total 56
-rw-r--r-- 1 bertacch piusers  923 Nov 25 11:47 multiHitFromChi2EDProducerDefault_cfi.py
-rw-r--r-- 1 bertacch piusers 1360 Nov 25 11:47 multiHitFromChi2EDProducerDefault_cfi.pyc
-rw-r--r-- 1 bertacch piusers  670 Nov 25 11:47 seedCreatorFromRegionConsecutiveHitsEDProducer_cfi.py
-rw-r--r-- 1 bertacch piusers 1062 Nov 25 11:47 seedCreatorFromRegionConsecutiveHitsEDProducer_cfi.pyc
-rw-r--r-- 1 bertacch piusers  692 Nov 25 11:47 seedCreatorFromRegionConsecutiveHitsTripletOnlyEDProducer_cfi.py
-rw-r--r-- 1 bertacch piusers 1095 Nov 25 11:47 seedCreatorFromRegionConsecutiveHitsTripletOnlyEDProducer_cfi.pyc
-rw-r--r-- 1 bertacch piusers  480 Nov 25 11:47 seedGeneratorFromL1TTracksEDProducer_cfi.py
-rw-r--r-- 1 bertacch piusers  831 Nov 25 11:47 seedGeneratorFromL1TTracksEDProducer_cfi.pyc
-rw-r--r-- 1 bertacch piusers 1015 Nov 25 11:47 SeedGeneratorFromProtoTracksEDProducer_cfi.py
-rw-r--r-- 1 bertacch piusers 1337 Nov 25 11:47 SeedGeneratorFromProtoTracksEDProducer_cfi.pyc
-rw-r--r-- 1 bertacch piusers  350 Nov 25 11:47 SeedingOTEDProducer_cfi.py
-rw-r--r-- 1 bertacch piusers  637 Nov 25 11:47 SeedingOTEDProducer_cfi.pyc
-rw-r--r-- 1 bertacch piusers  583 Nov 25 11:47 trackerClusterCheckDefault_cfi.py
-rw-r--r-- 1 bertacch piusers  926 Nov 25 11:47 trackerClusterCheckDefault_cfi.pyc
```